### PR TITLE
Installation2

### PR DESCRIPTION
**Pull request by slavapopkov**

### CODE REVIEW

**Comment by slavapopkov** [comment]
installation2

File: app/SymfonyRequirements.php [hunk 1]
```diff
@@ -0,0 +1,717 @@
+<?php
```
Comment (EduardTrayan) [inline app/SymfonyRequirements.php]: Please remove this file

File: app/check.php [hunk 2]
```diff
@@ -0,0 +1,142 @@
+<?php
```
Comment (EduardTrayan) [inline app/check.php]: Please remove this file

File: web/config.php [hunk 3]
```diff
@@ -0,0 +1,124 @@
+<?php
```
Comment (EduardTrayan) [inline web/config.php]: Please remove this file

File: app/SymfonyRequirements.php (hunk 1 above)
Comment (slavapopkov) [inline app/SymfonyRequirements.php]: ок

File: app/check.php (hunk 2 above)
Comment (slavapopkov) [inline app/check.php]: Ок

File: web/config.php (hunk 3 above)
Comment (slavapopkov) [inline web/config.php]: ок

**Pull request merged**